Formula/ski.rb

```diff
@@ -14,15 +24,18 @@ class Ski < Formula
   end
 
   head do
-    url "https://gitlab.com/esr/ski.git"
+    url "https://gitlab.com/esr/ski.git", branch: "master"
     depends_on "xmlto" => :build
   end
 
+  depends_on "python@3.10"
```

Comment:
Still open to suggestions on best way to handle `#!/usr/bin/env python3` dependency. Since it is compatible with `python` 2, it is possible to avoid dependency on macOS.

Using `python@3.10` directly will avoid another revision bump when `python` alias is migrated due to `rewrite_shebang`.

Using `uses_from_macos "python"` would reduce dependency tree.